### ISSUE TITLE
Improve Accessiblity of Cart Button

### ISSUE THREAD

**Issue by machineloop** [vendure-ecommerce/storefront-qwik-starter#18]
Need to change the markup for the Cart Button in upper right hand corner:

1. Add 'Cart' name attribute to cart button markup
2. Add aria-label containing something like `0 items in cart` to reflect count of added items to cart.

<img width="714" alt="Missing-Button-Name-Aria-label" src="https://user-images.githubusercontent.com/3682072/205052334-68aca693-2855-4ba1-ac5d-f53c7741c31b.png">

**From [MDN A11y docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#content_with_images_must_be_labeled):**

[Content with images must be labeled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#content_with_images_must_be_labeled)

Provide descriptive text for all contentful (that is, non-decorative) images and image-like elements. This includes SVG images, [<img>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img), [<canvas>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas), [<map>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map), and [<area>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/area) elements, as well as [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) elements where type=image and [<object>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object) elements where the type starts with image/. The typical way to do this is with the alt attribute. Be sure that the description conveys what is shown in the image.
[Example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#example_4)

<img
  src="milkweed.jgp"
  alt="Black and white close-up photo of milkweed flowers" />

[Interactive elements must be labeled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#interactive_elements_must_be_labeled)

If an element is intended for users to interact with it, it should have a label. Interactive elements include links ([<a>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)), form elements, buttons, and any element that has a handler for mouse or keyboard events. The way to label an element depends on its type: for form elements, use a [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label); for links, buttons and clickable elements, the text content of the element typically provides the label. If no other option exists for labeling an element, use the [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) attribute.

**Comment by machineloop** [comment]
I'm happy to take this one 👍
